dependencies.repos

```diff
@@ -1,14 +1,17 @@
 repositories:
-  BICA:
-    type: git
-    url: https://github.com/IntelligentRoboticsLabs/BICA.git
-    version: behavior_trees
   navigation2:
     type: git
     url: https://github.com/MROS-RobMoSys-ITP/navigation2.git
     version: pilot-urjc
-  ros2_planning_system:
-    type: git
-    url: https://github.com/IntelligentRoboticsLabs/ros2_planning_system.git
-    version: master
   
+  Pilot-URJC:
```

Comment:
@marioney I think that we shouldn't include as dependence the repo itself

Reply:
OK, I can change that. I just put it that way to have one less instruction.

Reply:
Solved in 77c43904922b516800c00c4ca219c4eec087f001